lib/js/jsre.nim

```diff
@@ -30,6 +30,18 @@ func newRegExp*(pattern: cstring): RegExp {.importjs: "new RegExp(@)".}
 func compile*(self: RegExp; pattern: cstring; flags: cstring) {.importjs: "#.compile(@)".}
   ## Recompiles a regular expression during execution of a script.
 
+func replace*(pattern: cstring; self: RegExp; replacement: cstring): cstring {.importjs: "#.replace(#,#)".}
+  ## Returns a new string with some or all matches of a pattern replaced by given replacement
+
+# func replaceAll*(pattern: cstring; self: RegExp; replacement: cstring): cstring {.importjs: "#.replaceAll(#,#)".}
+#   ## Returns a new string with all matches of a pattern replaced by given replacement
```

Comment:
Remove or fix commented out code.

Reply:
I initially thought `replaceAll` was already supported, but then I discovered it was just an ES2021 thing... so out it goes. :)

https://backbencher.dev/javascript/es2021-new-features